### PR TITLE
Add welcome modal to VAOS

### PR DESCRIPTION
**Pull request by jbalboni**
## Description
This adds a welcome modal to VAOS, using the announcement functionality.

## Testing done
Local testing

## Screenshots
Screen Shot 2020-03-05 at 11 30 24 AM

## Acceptance criteria
- [ ] Modal works

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/containers/AppointmentsPage.jsx [hunk 1]
```diff
@@ -18,18 +18,34 @@ import {
...
   componentDidMount() {
-    scrollAndFocus();
+    if (this.props.isWelcomeModalDimissed) {
+      scrollAndFocus();
+    }
     this.props.fetchFutureAppointments();
     document.title = `${pageTitle} | Veterans Affairs`;
   }
 
+  componentDidUpdate(prevProps) {
```
Comment (jbalboni) [inline src/applications/vaos/containers/AppointmentsPage.jsx]: Adding this to handle focusing after the modal is dismissed

**Comment by narin** [comment]
@jbalboni In this PR, could you please also add a GA event:`vaos-return-to-legacy-link-clicked`? Thanks.

**Review by narin — APPROVED** [review]
I'm not too familiar with the site-wide announcements but this LGTM.  How does localStorage come into play when tracking the dismissed annoucements?

**Comment by jbalboni** [comment]
@narin There's a `DISMISSED_ANNOUNCEMENTS` value in local storage that holds the keys for all the announcements that have been dismissed. When the announcements code starts up, it loads those values into Redux. It's all handled by that code, so we don't have to mess with it.

**Review by ncksllvn — APPROVED** [review]
LGTM!

**Review by jhonny218 — APPROVED** [review]
LGTM

**Pull request merged**